src/dashboard/src/installer/views.py

```diff
@@ -61,30 +61,22 @@ def welcome(request):
         archivematica_agent.save()
 
         # save organization PREMIS agent if supplied
-        org_name = request.POST.get('org_name', '')
-        org_identifier = request.POST.get('org_identifier', '')
-
-        if org_name != '' or org_identifier != '':
-            agent = Agent.objects.get(pk=2)
-            agent.name = org_name
-            agent.identifiertype = 'repository code'
-            agent.identifiervalue = org_identifier
-            agent.save()
-
-        # Save user and set cookie to indicate this is the first login
-        form = SuperUserCreationForm(request.POST)
+        form = OrganizationForm(request.POST)
         if form.is_valid():
-            user = form.save()
-            api_key = ApiKey.objects.create(user=user)
-            api_key.key = api_key.generate_key()
-            api_key.save()
-            user = authenticate(username=user.username, password=form.cleaned_data['password1'])
-            if user is not None:
-                login(request, user)
-                request.session['first_login'] = True
-                return redirect('installer.views.fprconnect')
+            org_name = request.POST.get('org_name', '')
+            org_identifier = request.POST.get('org_identifier', '')
+
+            if org_name != '' or org_identifier != '':
+                agent = Agent.objects.get(pk=2)
+                agent.name = org_name
+                agent.identifiertype = 'repository code'
+                agent.identifiervalue = org_identifier
+                agent.save()
+
+            request.session['first_login'] = True
+            return redirect('installer.views.fprconnect')
     else:
-        form = SuperUserCreationForm()
+        form = OrganizationForm()
```

Comment:
I've recently made some changes to `installer.views` in #4 that will cause some conflicts in this area. But it's simple code, we should be ok.